libutils/misc_lib.c

```diff
@@ -26,7 +26,9 @@
 
 #include <platform.h>
 #include <alloc.h>
+#include <string_lib.h>
 #include <logging.h>
+#include <sequence.h>
```

Comment:
I think this no longer needs `sequence.h`

Reply:
But I think you should `#include <errno.h>`

Reply:
Will make a follow up.